src/patternfly/components/Select/select.scss

```diff
@@ -188,48 +183,36 @@
     bottom: 0;
     left: 0;
     content: "";
-    border: var(--pf-c-select__toggle--BorderWidth) solid;
-    border-color: var(--pf-c-select__toggle--BorderTopColor) var(--pf-c-select__toggle--BorderRightColor) var(--pf-c-select__toggle--BorderBottomColor) var(--pf-c-select__toggle--BorderLeftColor);
+    border: var(--pf-c-select__toggle--before--BorderWidth) solid;
+    border-color: var(--pf-c-select__toggle--before--BorderTopColor) var(--pf-c-select__toggle--before--BorderRightColor) var(--pf-c-select__toggle--before--BorderBottomColor) var(--pf-c-select__toggle--before--BorderLeftColor);
   }
 
   &:hover {
     &::before {
-      --pf-c-select__toggle--BorderBottomColor: var(--pf-c-select__toggle--hover--BorderBottomColor);
+      --pf-c-select__toggle--before--BorderBottomColor: var(--pf-c-select__toggle--hover--before--BorderBottomColor);
     }
   }
 
```

Comment:
Looks like we're missing a `:focus` state - would you mind adding one in this PR?